### ISSUE TITLE
No documentation for custom Converter

### ISSUE THREAD

**Issue by newmanw** [square/retrofit#1305]
Would it be possible to add in some documentation on defining a custom converter with examples?

**Comment by JakeWharton** [comment]
What is missing? `Converter` and `Converter.Factory` have documentation, there's 6 built-in examples for 3 serialization formats, and the test code has a trivial example in `ToStringConverterFactory`.

**Comment by newmanw** [comment]
You are correct, there is enough there is you dig into the code.  However the main site provides examples for everything, without having to dig into the code.  Certainly not a huge deal at all, just thought it would be nice to see an example that accompanied this text.

<blockquote>
CUSTOM CONVERTERS

If you need to communicate with an API that uses a content-format that Retrofit does not support out of the box (e.g. YAML, txt, custom format) or you wish to use a different library to implement an existing format, you can easily create your own converter. Create a class that extends the Converter.Factory class and pass in an instance when building your adapter.
</blockquote>
